src/test/kotlin/org/objectionary/ddr/integration/resolver/ResolverBase.kt

```diff
@@ -43,78 +43,53 @@ import org.cactoos.io.ResourceOf
-    private val logger = LoggerFactory.getLogger(this.javaClass.name)
-    private val postfix = "tmp"
+open class ResolverBase : IntegrationTestBase {
+    override val logger = LoggerFactory.getLogger(this.javaClass.name)
+    override val postfix = "tmp"
+    private var testName: String = ""
+    private val xmirInTmp = "xmirs_tmp"
+    private val eoOutTmp = "eo_tmp"
 
     override fun doTest() {
-        val path = getTestName()
-        Files.walk(Paths.get(constructEoPath(path)))
-            .filter(Files::isRegularFile)
-            .forEach { eoToXMIR(it.toString()) }
-        constructInPath(path)
-        val documents = SrsTransformed(constructInPath(path), XslTransformer(), postfix).walk()
+        testName = getTestName()
+        eoToXmir(constructEoInPath(testName))
+        val sources = SrsTransformed(constructInPath(testName), XslTransformer(), postfix)
+        val documents = sources.walk()
         val graph = GraphBuilder(documents).createGraph()
         CondAttributesSetter(graph).processConditions()
         AttributesSetter(graph).setAttributes()
         InnerPropagator(graph).propagateInnerAttrs()
         CondNodesResolver(graph, documents).resolve()
         BasicDecoratorsResolver(graph, documents).resolve()
-        val cmpPath = constructCmpPath(path)
-        Files.walk(Paths.get(cmpPath))
-            .filter(Files::isRegularFile)
-            .forEach {
-                XslTransformer().singleTransformation(it.toString(), it.toString(), "/compress-aliases.xsl")
-                XslTransformer().singleTransformation(
-                    it.toString(),
-                    it.toString(),
-                    "/org/eolang/parser/wrap-method-calls.xsl"
-                )
-                XMIRToEo(it.toString(), path)
-            }
-        val outPath = constructOutPath(path)
-        val cmpFiles: MutableList<String> = mutableListOf()
-        Files.walk(Paths.get(outPath))
-            .filter(Files::isRegularFile)
-            .forEach { cmpFiles.add(it.toString()) }
-        val eoOutPath = constructEoOutPath(path)
-        Files.walk(Paths.get(eoOutPath))
+        xmirToEo(constructResultPath(testName))
+        Files.walk(Paths.get(constructOutPath(testName)))
             .filter(Files::isRegularFile)
             .forEach { file ->
-                val actualBr: BufferedReader = File(file.toString()).bufferedReader()
-                val actual = actualBr.use { it.readText() }
-                val expectedFile = cmpFiles.find {
-                    it.substringAfterLast(path) == file.toString().substringAfterLast(path)
-                }
-                val expectedBr: BufferedReader = File(expectedFile.toString()).bufferedReader()
-                val expected = expectedBr.use { it.readText() }
+                val expected = File(file.toString()).bufferedReader().use { it.readText().replace(" ", "") }
+                val actualFilename = file.toString().replace("$eoOutTmp$sep", "out$sep")
```

Comment:
@c71n93 We are trying to avoid compound names. According to: https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html